.eslintrc.json

```diff
@@ -3,9 +3,10 @@
 	"plugins": ["mocha", "chai-expect"],
 	"rules": {
 		"array-callback-return": "off",
+		"callback-return": "error",
 		"camelcase": "off",
-		"chai-expect/missing-assertion": 2,
-		"chai-expect/no-inner-compare": 2,
+		"chai-expect/missing-assertion": "error",
+		"chai-expect/no-inner-compare": "error",
```

Comment:
these 2 rules are enabled by default so deleting them would enable them. No need to keep these rules in eslintrc.json

Reply:
How do you know that? The readme says you can enable them as you like.
https://github.com/Turbo87/eslint-plugin-chai-expect#rules

Reply:
@diego-G you are right. I would expected them to be enabled by default by most of the eslint plugins do. Ignore this comment.